### PR TITLE
Feature a

### PR DESCRIPTION
**Pull request by Rilas2023**
added file1 and java1

### CODE REVIEW

**Comment by FerdaKizilay** [comment]
good doc.

**Comment by TatianaChise** [comment]
great job

File: src/java1.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+public class java1 {
+    //123
+
+
+    //456
+
+    //489
```
Comment (alexvinnyk) [inline src/java1.java:7]: 78

**Comment by alexvinnyk** [comment]
good

**Review by alexvinnyk — COMMENTED** [review]
good

**Review by alexvinnyk — COMMENTED** [review]
Practice

**Pull request merged**